server.py

```diff
@@ -0,0 +1,92 @@
+from bottle import route, run, template
+import requests
+import datetime
+from datetime import datetime, timedelta
+
+def get_date(date):
+    #2021-08-26
+    endpoint = f'https://www.recreation.gov/api/timedentry/availability/facility/10086745?date={date}'
+    headers = {'user-agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36'} 
+    return requests.get(endpoint, headers=headers)
+
+def check_count_for_date(date):
+    d = get_date(date)
+    if d:
+        val = d.json()
+        total_count = val[0]['inventory_count']['ANY']
+        reservation_count = val[0]['reservation_count']['ANY']
+        left = total_count - reservation_count
+        if left > 0:
+            return left
+    return None
+
+def get_resp_for_date(date):
+    result = f'[{date}] no :('
+    count = check_count_for_date(date)
+    if count:
+        result = f'[{date}] Yes, there are {count} tickets left'
+    return result
+
+@route('/check_date/<date>', method='POST')
+def check_date(date):
+    result = ''
+    if '|' in date:
+        dates = date.split('|')
+        if len(dates) == 2:
+            start = dates[0]
+            end = dates[1]
+
+            start = datetime.strptime(start, '%Y-%m-%d')
+            end = datetime.strptime(end, '%Y-%m-%d')
+
+            start_l = 0
+            end_l = 30
```

Comment:
What does `30` represent? If this is related to # of days in a month, should this be 31, or depend on the month?